packages/strapi-design-system/src/SimpleMenu/Menu.tsx

```diff
@@ -28,14 +28,15 @@ const MenuRoot = DropdownMenu.Root;
 interface TriggerProps extends ButtonProps {}
 
 const MenuTrigger = React.forwardRef<HTMLButtonElement, TriggerProps>(
-  ({ size, endIcon = <CarretDown width="0.6rem" height="0.4rem" aria-hidden />, ...props }, ref) => {
+  ({ size, endIcon = <CaretDown width="0.6rem" height="0.4rem" aria-hidden />, ...props }, ref) => {
     return (
       <DropdownMenu.Trigger asChild>
         <Button
           ref={ref}
           type="button"
           variant="ghost"
           endIcon={endIcon}
+          background="pink"
```

Comment:
Sorry if I'm pointing out things that are intended but a background="pink" default seems
strange here?

Reply:
No you're totally right, i think i was trying to test something out 😂